### PR TITLE
PI-816 fixed tests

### PR DESCRIPTION
**Pull request by thellimist**

### CODE REVIEW

**Comment by jwineman** [comment]
What changed here?  The diff makes it look like the entire file was added/removed.

File: src/Test/WordPress/PluginActionsTest.php [hunk 1]
```diff
@@ -2,155 +2,154 @@
...
+                        'legacy_id' => Plans::FREE_PLAN,
+                    ),
+                ),
+            )
+        );
+        $this->mockWordPressClientAPI->method('responseOk')->willReturn(true);
+        $this->mockWordPressClientAPI->method('changeZoneSettings')->willReturn(true);
+        $this->mockWordPressClientAPI->method('createPageRule')->willReturn(true);
+
+        $this->mockWordPressClientAPI->expects($this->exactly(14))->method('changeZoneSettings');
```
Comment (thellimist) [inline src/Test/WordPress/PluginActionsTest.php]: This

File: src/Test/WordPress/PluginActionsTest.php [hunk 2]
```diff
@@ -2,155 +2,154 @@
...
+                        'legacy_id' => Plans::BIZ_PLAN,
+                    ),
+                ),
+            )
+        );
+        $this->mockWordPressClientAPI->method('responseOk')->willReturn(true);
+        $this->mockWordPressClientAPI->method('changeZoneSettings')->willReturn(true);
+        $this->mockWordPressClientAPI->method('createPageRule')->willReturn(true);
+
+        $this->mockWordPressClientAPI->expects($this->exactly(16))->method('changeZoneSettings');
```
Comment (thellimist) [inline src/Test/WordPress/PluginActionsTest.php]: This

**Comment by thellimist** [comment]
@jwineman Commented the changes. #linterthings

**Pull request merged**